airflow/providers/openlineage/utils/sql.py

```diff
@@ -155,11 +155,17 @@ def create_information_schema_query(
     metadata = MetaData(sqlalchemy_engine)
     select_statements = []
     for db, schema_mapping in tables_hierarchy.items():
-        schema, table_name = information_schema_table_name.split(".")
         if db:
-            schema = f"{db}.{schema}"
+            schema = db
```

Comment:
Does this mean `schema` is `database schema` in Oracle meaning here, not `schema of a database`? Would be nice to document this if true - this is cause of many errors usually.

Reply:
That is a technical solution related to how SQLAlchemy constructs table identifiers. The assumption is to have `database` -> `schema` -> `table_name` hierarchy with first two optional.
I've added a short explanation how above solution works.

Reply:
👍